### ISSUE TITLE
Incompatibility between sakai and angular 18 !!!!

### ISSUE THREAD

**Issue by JulieRaymond** [primefaces/sakai-ng#55]
Unfortunately, it is impossible to make sakai run with angular 18.

**Comment by devshack** [comment]
> Unfortunately, it is impossible to make sakai run with angular 18.

It works for me using Angular 19, what exactly is your issue?

**Comment by JulieRaymond** [comment]
Perfect, i have found a video on youtube explaining how to pair Sakai template from PrimeNg to angular 18. Works perfectly :)

**Comment by francesco-rossetti** [comment]
@JulieRaymond can you post which video did you follow?

**Comment by JulieRaymond** [comment]
@francesco-rossetti of course, this one : https://www.youtube.com/watch?v=Kj-83bdXq8M&list=PLHbvUzceyvgH8njtR69DZdWzFJ2BiVa6P&index=5

Few others from the same owner are really good to understand better sakai, and primeNg :)

**Comment by francesco-rossetti** [comment]
> @francesco-rossetti of course, this one : https://www.youtube.com/watch?v=Kj-83bdXq8M&list=PLHbvUzceyvgH8njtR69DZdWzFJ2BiVa6P&index=5
> 
> Few others from the same owner are really good to understand better sakai, and primeNg :) 

Thanks a lot!
